privacyidea/static/components/token/views/token.details.html

```diff
@@ -71,7 +71,14 @@ <h3 name="Token Details"><translate>Token details for {{ tokenSerial }}
         <tr ng-if="token.rollout_state">
             <td translate>Rollout State</td>
             <td>{{ token.rollout_state }}</td>
-            <td></td>
+            <td>
+                <div ng-show="token.rollout_state === 'verify'">
+                    <input type="text" ng-model="token.verifyResponse" class="form-control"
+                           placeholder="{{ 'Enter code to verify enrollment' | translate }}">
+                    <button type="button" ng-click="sendVerifyResponse()"
+                            class="btn btn-primary" translate>Verify Token</button>
+                </div>
+            </td>
```

Comment:
I like this :-)

Reply:
Yes, when I realized that it was fairly simple to verify the enrollment in the token details I was happy, too.